Trappist/src/Promact.Trappist.Repository/Test/ITestsRepository.cs

```diff
@@ -26,13 +28,6 @@ public interface ITestsRepository
         Task<List<Test>> GetAllTestsAsync();
 
         /// <summary>
-        /// Gets the settings saved of a particular Test
-        /// </summary>
-        /// <param name="id">The parameter "id" is used to get the Settings of a Test by its Id</param>
-        /// <returns>Settings Saved for the selected Test</returns>
-        Task<Test> GetTestByIdAsync(int id);
-
-        /// <summary>
         /// Updates the changes made to the settings of a Test
         /// </summary>
         /// <param name="testObject">The parameter "testObject" is an object of Test</param>
```

Comment:
Mam, I need this object to update the fields of Test Settings Page in database. So, it is to be kept.